### PR TITLE
fix: divide value by 100 vo cut the cents

### PR DESCRIPTION
**Pull request by dariakoko**
We used cants as if it was dollars, which caused the Annual Value Sold to be 100 times bigger than it should be.

### CODE REVIEW

**Review by olerichter00 — DISMISSED** [review]
Thanks a lot for the fix 🙏

**Comment by MounirDhahri** [comment]
Let's merge this as soon as we fix the broken tests

File: src/schema/v2/artwork/index.ts [hunk 1]
```diff
@@ -110,7 +110,7 @@ const ArtworkPriceInsightsType = new GraphQLObjectType<any, ResolverContext>({
       type: GraphQLString,
       description: 'The annual value of the work sold "in USD "',
       resolve: ({ annualValueSoldCents }) => {
-        return `$${formatLargeNumber(annualValueSoldCents)}`
+        return `$${formatLargeNumber(annualValueSoldCents / 100)}` // dividing by 100 because we don't need the value in cents
```
Comment (joeyAghion) [inline src/schema/v2/artwork/index.ts:113]: We should avoid doing currency math in multiple, local, view-oriented places. Better touse the money helpers that are (I think) already available in this project, and avoid currency differences, rounding, floating-point precision issues, etc.

Comment (sweir27) [reply src/schema/v2/artwork/index.ts:113]: +1 to this! Baking in `/ 100` makes it much harder to support different currencies in the future.

Comment (MounirDhahri) [reply src/schema/v2/artwork/index.ts:113]: @dariakoko let's fix this and use the money helper then.

**Pull request merged**